core/cli/src/install.ts

```diff
@@ -112,11 +113,13 @@ export default async function installHooks(logger: Logger): Promise<ValidConfig>
     }
   }
 
+  await runInit(logger, config)
+
   const errors: Error[] = []
-  // group hooks without an installGroup separately so that their check()
-  // method runs independently
```

Comment:
this comment got moved away from the thing it was commenting on at some point, so i've just moved it back